default.nix

```diff
@@ -1,10 +1,10 @@
-{ pkgs ? import (fetchTarball "https://github.com/NixOS/nixpkgs/archive/4c3edba85629ec304b5269e4b0ac7f26c433df23.tar.gz") { } }:
+{ pkgs ? import (fetchTarball "https://github.com/NixOS/nixpkgs/archive/f2406198ea0e4e37d4380d0e20336c575b8f8ef9.tar.gz") { } }:
 
 pkgs.mkShell {
   buildInputs = [
     pkgs.nil
     pkgs.nixpkgs-fmt
-    pkgs.nodejs-19_x
+    pkgs.nodejs_20
```

Comment:
Using flake.nix may drop this manual sync with .node-version, but I'm a nobie of the nix ecosystem... 🤷‍♂️